opennms-doc/guide-admin/src/asciidoc/text/operation/cassandra21x.adoc

```diff
@@ -0,0 +1,112 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../images
+=== Cassandra monitoring
+
+This section describes some of the metrics _OpenNMS_ collects for monitoring an _Apache Cassandra_ database
+infrastructure. The category `Cassandra21x` is used to enable the data collection on a node. Also, the service
+name `Cassandra21x` is used to bind this data collection to a node's interface. The _JMXCollector_ is used
+to retrieve the _MBean_ entities from the _Cassandra_ node.
+
```

Comment:
Can you please make one sentence per line?
Than it is easier to review and also document changes are better "diffed" in the future.
Please see the official opennms documentation guide for more information: http://docs.opennms.org/opennms/branches/develop/guide-doc/guide-doc.html.